ui/src/app/shared/services/workflows-service.ts

```diff
@@ -31,7 +31,8 @@ export class WorkflowsService {
             'items.metadata.labels',
             'items.status.phase',
             'items.status.finishedAt',
-            'items.status.startedAt'
+            'items.status.startedAt',
+            'items.spec'
```

Comment:
this is a lot of data - can we avoid it?

Reply:
I'll check if it can be further specified. I believe I added that for the Util functions for checking to see if an action can be performed on a wf. I'll also see if that can be determined without using `spec` at all.

Reply:
Only needed `item.spec.suspended` for the check. Just fixed